hazelcast-sql/src/test/java/com/hazelcast/sql/SqlBasicTest.java

```diff
@@ -572,7 +572,7 @@ static String portableFieldName(String fieldName) {
         return fieldName + "_p";
     }
 
-    abstract static class AbstractPojoKey implements Serializable {
+    abstract static class AbstractPojoKey implements Serializable, Comparable {
```

Comment:
Why it has to be `Comparable` suddenly?

Reply:
Had a bug, fix it, but haven't remove `Comparable`. Thanks.